packages/localize-tools/src/program-analysis.ts

```diff
@@ -517,7 +517,8 @@ function replaceHtmlWithPlaceholders(
 
   const traverse = (node: ChildNode): void => {
     if (node.nodeName === '#text') {
-      const text = (node as TextNode).value;
+      const location = node.sourceCodeLocation!;
+      const text = html.substring(location.startOffset, location.endOffset);
```

Comment:
How is this different form `node.value`? Is it because it preserves the escape sequences? If so, a comment will help future editors.